ui/src/components/error/index.css

```diff
@@ -0,0 +1,32 @@
+.error-container {
+  background-color: #fdd;
+  padding: 16px;
+  border-radius: 8px;
+  border: 1px solid #f88;
+  text-align: center;
+  max-width: 500px;
+  margin: 20px auto;
+
+  h2 {
+    font-size: 18px;
+    font-weight: bold;
+    color: #d44;
+    margin-bottom: 12px;
+  }
+}
+
+.error-message {
+  color: #d66;
+  white-space: pre-wrap;
+  margin-bottom: 12px;
+}
+
+.home-button,
+.language-button {
```

Comment:
Could you make these names more specific? Maybe stick `error-fallback-` to the front of each. Stylesheets are applied globally once loaded, so you have to be careful about name collisions when using more generic class names.

Reply:
Thanks. I've pushed the changes. 

Merging the branch.